src/main/java/com/github/monster860/fastdmm/FastDMM.java

```diff
@@ -805,4 +870,59 @@ private void loop() {
 			Display.sync(60);
 		}
 	}
+
+	private String getAppData() {
```

Comment:
So, from what I understand, you actually want me to create a folder in the user's root directory on Windows? That doesn't seem like a good idea to me.